drillsrs/cmd/import_.py

```diff
@@ -70,11 +106,21 @@ def decorate_arg_parser(self, parser: argparse.ArgumentParser) -> None:
             nargs="?",
             help="path to import from; if omitted, standard input is used",
         )
+        parser.add_argument(
+            "-a",
+            action="store_true",
+            help="Import Anki Deck",
+        )
 
     def run(self, args: argparse.Namespace) -> None:
         path: Optional[str] = args.path
+        anki: Optional[bool] = args.a
```

Comment:
After adding `--anki` above you can now refer to this option as `args.anki` which is more readable